Formalisation(shacl)/Core/PiecesShape/Distribution.ttl

```diff
@@ -230,5 +231,6 @@ _:a07deeba86ea4a158f0b02eb8017e72a5 rdf:first status:DEPRECATED;
   sh:description "This property can be repeated for parallel language versions of the description."@en;
   sh:minCount 1;
   sh:nodeKind sh:Literal;
+  sh:uniqueLang true;
```

Comment:
This is on `title`, not description; the description for this should reflect that.